signalfx-org-metrics/metrics.yaml

```diff
@@ -1799,3 +1799,160 @@ sf.org.ui.num.pagevisits:
   description: Reserved for Infrastructure Monitoring internal use only.
   metric_type: null
   title: sf.org.ui.num.pagevisits
+
+sf.org.rum.numSpansReceived:
+  brief: The number of spans Splunk RUM accepts after filtering and throttling. 
+  description: |
+    Number of spans received. 
+
+    * Dimension: orgId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpansReceived
+
+sf.org.rum.numSpansReceivedByToken:
+  brief: The number of spans Splunk RUM received for a specific access token
+    after filtering and throttling.
+  description: |
+    The number of spans Splunk RUM received for a specific access token
+    after filtering and throttling.
+
+    * Dimension: orgId, tokenId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numSpansReceivedByToken
+
+
+sf.org.rum.numAddSpansCalls:
+  brief:  The number of calls to the `/v2/trace` endpoint.
+  description: |
+    The number of calls to the `/v2/trace` endpoint.
+
+    * Dimension: orgId
+    * Data resolution: 10 seconds
+  metric_type: Counter
+  title: sf.org.rum.numAddSpansCalls
+
+sf.org.rum.numAddSpansCallsByToken:
+ brief:  The number of calls to the ... endpoint for a specific access token.
```

Comment:
hi! @atoulme :) which endpoint is correct ? thanks!

Reply:
I'm sorry, I don't know this. A member of the RUM team can maybe help you here.

Reply:
got it, thanks!